content/en/blog/2019/monitoring-external-service-traffic/index.md

```diff
@@ -0,0 +1,373 @@
+---
+title: "Monitoring blocked and passthrough external service traffic"
+description: "How can you use Istio to monitor blocked and passthrough external traffic."
+publishdate: 2019-09-23
+attribution: Neeraj Poddar (Aspen Mesh)
+keywords: [monitoring,blackhole,passthrough]
+---
+
+## Monitoring blocked and passthrough external service traffic
```

Comment:
You shouldn't start a page with a header, since there's already a header inserted automatically. So just remove this line.